src/pybind/mgr/dashboard/controllers/cephfs.py

```diff
@@ -101,6 +101,29 @@ def rename(self, name: str, new_name: str):
                 component='cephfs')
         return f'Volume {name} renamed successfully to {new_name}'
 
+    @UpdatePermission
+    @Endpoint('PUT')
+    @EndpointDoc("Set Ceph authentication capabilities for the specified user ID in the given path",
+                 parameters={
+                     'fs_name': (str, 'File system name'),
+                     'client_id': (str, 'Cephx user ID'),
+                     'caps': (str, 'Path and given capabilities'),
+                     'root_squash': (str, 'File System Identifier'),
+
+                 })
+    def auth(self, fs_name: str, client_id: int, caps: List[str], root_squash: bool):
+        if root_squash:
+            caps.insert(2, 'root_squash')
+        error_code, _, err = mgr.mon_command({'prefix': 'fs authorize',
+                                              'filesystem': fs_name,
+                                              'entity': client_id,
+                                              'caps': caps})
+        if error_code != 0:
+            raise DashboardException(
+                msg=f'Error setting authorization for {client_id} with {caps}: {err}',
+                component='cephfs')
+        return f'Updated {client_id} authorization successfully'
```

Comment:
`Created Ceph user "client.xx" with required authorization successfully`

![image](https://github.com/ceph/ceph/assets/6695156/26f49cab-c6a8-43c9-a88b-2455b6f1f6e5)

Reply:
User can be created if it doesn't exist or updated if it does. That's why I wrote down a generic message.

Not sure if you want any other changes to the message